modules/travel_pay/app/controllers/travel_pay/v0/claims_controller.rb

```diff
@@ -6,7 +6,7 @@ class ClaimsController < ApplicationController
       def index
         begin
           token_service.get_tokens(@current_user) => { veis_token:, btsss_token: }
-          claims = claims_service.get_claims(veis_token, btsss_token)
+          claims = claims_service.get_claims(veis_token, btsss_token, params)
```

Comment:
Do we want to move the `get_tokens` call to happen in the `claims_service` so all we pass in is the user rather than both tokens? 🤔

Reply:
Yes we do. I'm saving that for another PR unless you feel very strongly about doing it here.